react/components/EXPERIMENTAL_TableTree/Tree/Arrow.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react'
+
+import CaretUp from '../../icon/CaretUp/index.js'
+import CaretDown from '../../icon/CaretDown/index.js'
+import ButtonWithIcon from '../../ButtonWithIcon/index.js'
+
+const Arrow: FC<ArrowProps> = ({ active, onClick }) => {
+  const icon = active ? <CaretUp /> : <CaretDown />
+  return (
+    <span className="ph2">
+      <ButtonWithIcon
+        size="small"
+        onClick={onClick}
+        icon={icon}
+        variation="tertiary"
+      />
+    </span>
+  )
+}
+
+export type ArrowProps = {
+  active: boolean
+  onClick: Function
+}
+
+export default Arrow
```

Comment:
Could this be rather added to the icon's list?

Reply:
Nope. This component is a Button that toggles collapse on each row and uses two icons (arrow up and down). The name was changed to `CollapseToggle` to avoid confusion.